examples/build.vpc.centos-parallel.pkr.hcl

```diff
@@ -1,12 +1,16 @@
 packer {
   required_plugins {
     ibmcloud = {
-      version = ">=v3.0.0"
+      version = ">=v3.0.4"
       source = "github.com/IBM/ibmcloud"
     }
   }
 }
 
+variable "ENCRYPTION_KEY_CRN" {
+  type = string
+}
+
```

Comment:
ENCRYPTION_KEY_CRN is not being used anywhere. Do we need this here. Same for all files except encryption example

Reply:
It was kept as a placeholder, will remove it. Make sense to just keep it at one example.